File: app/src/app/dto/school/school-dto.ts
import { EntityUrl } from "src/app/enum/entity";
import { BaseDto } from "../utilities/base-dto";

export class SchoolDto extends BaseDto {
  id: number;
  name: string;
  adress: string;
  town: string;
  description: string;
  constructor(typeInstance?: string) {
    if (typeInstance) {
      super(typeInstance, EntityUrl.Schools);
    } else {
      super('SchoolDto', EntityUrl.Schools);
    }
    this.id = 0;
    this.name = '';
    this.adress = '';
    this.town = '';
    this.description = '';
  }
}

File: api/ManagementProject.Application/Features/Schools/SchoolDto.cs
using ManagementProject.Application.Features.Dto;

namespace ManagementProject.Application.Features.Schools
{
    public record SchoolDto : IBaseDto
    {
        public long Id { get; set; }
        public string? Name { get; set; }
        public string? Adress { get; set; }
        public string? Town { get; set; }
        public string? Description { get; set; }
    }
}
